natlas-server/migrations/migrator.py

```diff
@@ -0,0 +1,43 @@
+from alembic.migration import MigrationContext
+from alembic import context
+from alembic.config import Config as alembic_cfg
+from alembic.script import ScriptDirectory
+import flask_migrate
+import sqlalchemy
+
+
+def migration_needed(sqlalchemy_uri):
+    try:
+        engine = sqlalchemy.create_engine(sqlalchemy_uri)
+        conn = engine.connect()
+    except sqlalchemy.exc.OperationalError:
+        raise
```

Comment:
This seems redundant. Why would we wrap this in a try-catch if we're just going to throw it?

Reply:
You're right, there used to be other code here earlier in dev and I pulled it out and forgot to change this